bpf/kmesh/ads/include/ctx/sock_addr.h

```diff
@@ -21,4 +21,6 @@ typedef struct bpf_sock_addr ctx_buff_t;
     (ctx)->user_ip4 = (address)->ipv4;                                                                                 \
     (ctx)->user_port = (address)->port
 
+#define MARK_REJECTED(ctx)
```

Comment:
not sure what is this for

Reply:
cgroups/connect4 and sockops both use cluster_manager, without it will compile fail